make_wordvectors.sh

```diff
@@ -14,7 +14,7 @@ echo "step 0. Make `data` directory and move there.`
 mkdir data; cd data
 
 echo "step 1. Download the stored wikipedia file to your disk."
-wget "https://dumps.wikimedia.org/${lcode}wiki/20161201/${lcode}wiki-20161201-pages-articles-multistream.xml.bz2"
+wget "https://dumps.wikimedia.org/${lcode}wiki/latest/${lcode}wiki-latest-pages-articles.xml.bz2"
 
 echo "step 2. Extract the bz2 file."
 bzip2 -d "${lcode}wiki-20161201-pages-articles-multistream.xml.bz2"
```

Comment:
Probably helpful to change line 20's [`wiki-2016201`]() to [`wiki-latest`]() or whatever it becomes upon downloading locally